src/Sulu/Bundle/AdminBundle/Resources/js/components/Popover/Popover.js

```diff
@@ -45,12 +51,19 @@ export default class Popover extends React.PureComponent<Props> {
         window.removeEventListener('resize', this.close);
     }
 
-    componentDidUpdate(prevProps: Props) {
-        if (this.popoverChildRef && !prevProps.open && this.props.open) {
-            afterElementsRendered(() => {
-                this.popoverChildRef.scrollTop = this.dimensions.scrollTop;
-            });
+    componentDidUpdate() {
+        if (!this.popoverChildRef) {
+            return;
         }
+
+        afterElementsRendered(() => {
+            this.popoverChildRef.scrollTop = this.dimensions.scrollTop;
+
+            if (this.shouldUpdateDimensions) {
```

Comment:
Why do we need this variable for that? Why isn't this `afterElementsRendered` part happening right in the `triggerResize` method?

Reply:
The `popoverChildRef` refers to an old element at that point. Deferring the `updateDimensions` makes the function use the current `popoverChildRef`. Maybe that would also work by making `popoverChildRef` an observable but I have to try that and I am not sure if that would be performance wise a better solution.